packages/puppeteer-core/package.json

```diff
@@ -172,5 +172,8 @@
     "tar-fs": "2.1.1",
     "unbzip2-stream": "1.4.3",
     "ws": "8.11.0"
+  },
+  "peerDependencies": {
```

Comment:
I think peer dependency might print a warning/fail install if not satisfied. Since not all users use TypeScript I don't think we should include it.